scripts/spark-container.dockerfile

```diff
@@ -30,7 +30,7 @@ RUN dnf install -y java-11-openjdk less procps python3 tzdata \
     && ln -s $(which python3) /usr/bin/python \
     && tar -zxf $SPARK_PKG \
     && mv ./spark-${SPARK_VERSION}-bin-hadoop3 $SPARK_HOME \
-    && mv ./$SPARK_CONNECT_JAR ${SPARK_HOME}/jars/
+    && mv ./$SPARK_CONNECT_JAR /opt/
```

Comment:
@rui-mo, `SPARK_HOME` is an env variable. Can't you set the value when building the image?

Reply:
I guess `SPARK_CONNECT_JAR` cannot live under any `${SPARK_HOME}/jars/`.

Reply:
I feel a portable workaround is to use `${SPARK_HOME}/misc/`? Is this valid?

Reply:
Thanks for your review. You are right. We need to move `SPARK_CONNECT_JAR` out from `${SPARK_HOME}/jars/`.

> I feel a portable workaround is to use ${SPARK_HOME}/misc/. Is this valid?

I tried it locally and it also works. Updated this PR following your suggestion. Thanks.